frontend/src/styles/vars.scss

```diff
@@ -184,6 +184,7 @@ $_lifecycle_dormant: $_danger;
 
     // Non-color vars
     --radius: 4px;
+    --shadow: 0 20px 25px -5px rgba(0, 0, 0, 0.1), 0 8px 10px -6px rgba(0, 0, 0, 0.1);
     --shadow-elevation: 0px 16px 16px -16px rgba(0, 0, 0, 0.35);
```

Comment:
Why not reuse our current shadow? It will be very hard for any dev to say which variant to use